modules/client/test/integration/client-addressList/encoding.test.ts

```diff
@@ -42,10 +42,10 @@ describe("Client Address List", () => {
       const ctxPlugin = ContextPlugin.fromContext(ctx);
       const client = new ClientAddressList(ctxPlugin);
 
-      const pluginConfigParams: IPluginSettings = {
+      const pluginConfigParams: VotingSettings = {
         minDuration: 100000,
-        minTurnout: 0.25,
-        minSupport: 0.51,
+        minParticipation: 0.25,
+        supportThreshold: 0.51,
```

Comment:
Is the 0.51 value there for a specific reason? Now, these 2 values have changed how they work:
- `minParticipation` is checked with `>=`
- `supportThreshold` is checked with `>`

Reply:
no, is an arbitrary value that happens to be 0.51

Reply:
Well, if we are testing the SDK with certain values where the comparison operand could be an issue, then let's check it. Otherwise, I'm happy to resolve the comment

Reply:
we are not, so lets resolve